jsonschema-core.xml

```diff
@@ -1830,6 +1830,48 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "dereferenced" or
+                    "bundled" schema) which has multiple embedded JSON Schema Resources bundled into the same
+                    document to ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        A Compound Schema Document is created by replacing references (such as "$ref") with an
+                        embedded version of the referenced Schema Resource.
+                    </t>
+                    <t>
+                        Each embedded JSON Schema Resource (and the document's root) MUST identify itself
+                        with an absolute URI using the "$id" keyword, and SHOULD make use of the "$schema"
+                        keyword to identify the dialect it is using, in the root of the schema resource.
+                    </t>
+                    <t>
+                        When a reference applicator is dereferenced as part of the bundling process, the Schema Resource
+                        MUST NOT be embedded by replacing the schema object from which it was referenced, or by wrapping
```

Comment:
This is where clarity on "dereference" vs "bundling" would be helpful.  If "bundling" is _defined_ as embedding under `$defs` (transitively until all references can be satisfied within the document), then this is not a concern for bundling.  It's just not part of that process.

**Separately**, we can discuss dereferencing which might be applied after bundling, or in various other contexts.  I think that might add clarity.  Bundling should be really, really easy to implement.  Dereferencing is substantially more complex.

Reply:
I still think there's still confusion around "dereferencing" and "bundling" (and reference removal) here.

* _dereferencing_, elsewhere in this document, is about following references during evaluation
* _bundling_ is simply inspecting references and to figure out which resources need to be placed in a single document- there's no dereferencing going on at all
* _reference removal_ is when you actually take the target of a reference and inline it; this is not always possible, and I don't think is something we really want to discuss.

Reply:
Yeah, you're right, I've got my termonology muddled, and it might be that I wrote that in part confusion about how it would work anyway.
Lemme re-work!

Reply:
Done. You'll see some changes here. No longer used "dereferencing".